Samples/tutorials/outlook-tutorial/README.md

```diff
@@ -0,0 +1,92 @@
+---
+page_type: sample
+urlFragment: office-outlook-add-in-tutorial
+products:
+- office-outlook
+- m365
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: 09/12/2023 4:00:00 PM
+description: "A completed version of the step-by-step Outlook tutorial hosted on learn.microsoft.com."
+---
+
+# Outlook Tutorial - Completed
+
+## Summary
+
+This sample is the result of completing the [Tutorial: Build a message compose Outlook add-in](https://learn.microsoft.com/office/dev/add-ins/tutorials/outlook-tutorial). It was constructed with the [Yeoman generator for Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/develop/yeoman-generator-overview).
+
+The tutorial gives step-by-step instructions on how to add functionality alongside explanations as to why code is being added. Use this sample if you want to explore and try the completed code, or if you need to debug any issues you may have encountered while following the tutorial.
+
+## Features
+
+This sample features the basics of working with a compose message in Outlook. The functions collect information from the user, fetch data from an external service, implement a function command, and implement a task pane that inserts content into the body of a message. The sample also shows how to use a dialog box.
```

Comment:
Since "Features" is the header, it seemed jarring to me to have "features" used in the first sentence with a different meaning.
```suggestion
This sample demonstrates the basics of working with a compose message in Outlook. The functions collect information from the user, fetch data from an external service, implement a function command, and implement a task pane that inserts content into the body of a message. The sample also shows how to use a dialog box.
```